src/Microsoft.Identity.Client/Internal/PlatformPlugin.cs

```diff
@@ -61,8 +61,8 @@ public static void InitializeByAssemblyDynamicLinking()
         {
 #if !FACADE
             InjectDependecies(
-                (IWebUIFactory) new WebUIFactory(),
-                (ILogger) new PlatformLogger(),
+                (IWebUIFactory) new Microsoft.Identity.Client.Internal.WebUIs.WebUIFactory(),
```

Comment:
consider adding using statement to consolidate namespace usage. This comment applies to all other files as well.